### ISSUE TITLE
SGW related Question

### ISSUE THREAD

**Issue by jammer345** [tvayer/SGW#1]
I have a question to ask: is it possible to use SGW in GANs using DCGAN architecture by minimizing the SGW loss?

**Comment by tvayer** [comment]
Yes it is possible and would be interesting to see the results !
